ui/src/components/Common/Demo/Demo.vue

```diff
@@ -36,60 +50,61 @@
   <hr />
 
   <p>
-    <Button    
-    :label="buttonLabel"
-    primeClass="sm" icon="pi pi-users" 
-    badge="8"
-    ></Button>
+    <Button :label="buttonLabel" primeClass="sm" icon="pi pi-users" badge="8"></Button>
   </p>
 
   <hr />
   <h5>Basic Radio button</h5>
-    <div class="p-field-radiobutton">
-        <RadioButton id="city1" name="city" value="Chicago" v-model="city" :disabled=true />
-        <label for="city1">Chicago</label>
-    </div>
-    <div class="p-field-radiobutton">
-        <RadioButton id="city2" name="city" value="Los Angeles" v-model="city" />
-        <label for="city2">Los Angeles</label>
-    </div>
-    <div class="p-field-radiobutton">
-        <RadioButton id="city3" name="city" value="New York" v-model="city" />
-        <label for="city3">New York</label>
-    </div>
-    <div class="p-field-radiobutton">
-        <RadioButton id="city4" name="city" value="San Francisco" v-model="city" />
-        <label for="city4">San Francisco</label>
-    </div>
+  <div class="p-field-radiobutton">
+    <RadioButton id="city1" name="city" value="Chicago" v-model="city" :disabled="true" />
+    <label for="city1">Chicago</label>
+  </div>
+  <div class="p-field-radiobutton">
+    <RadioButton id="city2" name="city" value="Los Angeles" v-model="city" />
+    <label for="city2">Los Angeles</label>
+  </div>
+  <div class="p-field-radiobutton">
+    <RadioButton id="city3" name="city" value="New York" v-model="city" />
+    <label for="city3">New York</label>
+  </div>
+  <div class="p-field-radiobutton">
+    <RadioButton id="city4" name="city" value="San Francisco" v-model="city" />
+    <label for="city4">San Francisco</label>
+  </div>
   <hr />
-<label>selected radio button values : {{city}}</label>
+  <label>selected radio button values : {{ city }}</label>
 
-<hr />
-<div>
-  <RadioButtonGroup name="cityGroup" :radioValues="categories" @selectedRadioVal="selectedValue"/>
-  <label>selected radio button values : {{cityGroupName}}</label>
-</div>
+  <hr />
+  <div>
+    <RadioButtonGroup name="cityGroup" :radioValues="categories" @selectedRadioVal="selectedValue" />
+    <label>selected radio button values : {{ cityGroupName }}</label>
+  </div>
 </template>
 
 <script setup lang="ts">
 
 import { ref } from 'vue'
 import InputText from '../InputText.vue'
+import InputNumber from '../InputNumber.vue'
 import DropDown from '../DropDown.vue'
 import FieldSet from '../FieldSet.vue';
 import TimePicker from '../TimePicker.vue'
 import Button from '../Button.vue';
 import RadioButton from '../RadioButton.vue';
 import RadioButtonGroup from '../RadioButtonFieldSet.vue';
 
+const minVal = ref(5);
+const maxVal = ref(10);
+const stepVal = ref(2);
```

Comment:
Unless these values are intended to be updated somewhere, we won't need to wrap them in a ref.
 eg: `const minVal = 5` works

Reply:
Removed these const values as of now.